.github/workflows/publishToAzure.yml

```diff
@@ -33,6 +33,6 @@ jobs:
       uses: fixpoint/azblob-upload-artifact@v4
       with:
         connection-string: ${{ secrets.ConnectionString }}
-        name: app.publish
-        container: testContainer
-        path: ./bin/src/IssueCreator/Debug/app.publish
+        name: 'installer/..'
```

Comment:
This resolves to just ".". Or is this not supposed to resolve at all? Just that th ename is "installer/.."?

Reply:
It was intended to have it resolve to that because the current installer is at the root of the container. Without this hack we would have had to ask everyone to re-install the application.